server/index.js

```diff
@@ -4,7 +4,7 @@ const port = 8080;
 const app = express();
 
 console.log(__dirname);
-app.use(express.static(path.join(__dirname, '/../client/dist')));
```

Comment:
is the second backslash not needed when you use path.join?

Reply:
i was playing around with it because i like to try to use minimal code if possible. seems like it's not required.

Reply:
cool good to know!